examples/certificates/key.pem

```diff
@@ -1,5 +1,5 @@
 -----BEGIN PRIVATE KEY-----
-MIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQga38//86g0SxU++MI
-USzwolTXEHJcI8mHE9q1PL1HnEihRANCAAQbzFFi0ouIIjojztfw7TkVmjBx/DtS
-tUDM/TvggHWbrthlHhy91hpGwZtn+87da6QhjTzgjMIlccOzItriocOB
+MIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQg519TDzD9PTgZndCB
```

Comment:
we should remove this file completely and only ever have the user generate their own